.github/workflows/change-file-in-pr.yml

```diff
@@ -0,0 +1,30 @@
+name: Change File Included in PR
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, labeled]
+
+jobs:
+  check-files-in-directory:
+    if: ${{ !contains(github.event.pull_request.labels.*.name, 'Release Not Needed') }}
+    name: Change File Included in PR
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout PR code
+        uses: actions/checkout@v3
+
+      - name: Get List of Changed Files
+        id: changed-files
+        uses: tj-actions/changed-files@v34
```

Comment:
Looking at https://github.com/marketplace/actions/changed-files, the latest version is `v45.0.4`. I think we should pin this to the corresponding commit SHA and use that.

Reply:
Done